src/xdocs/checks/coding/requirethis.xml.template

```diff
@@ -61,41 +61,41 @@
         </p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/Example1.txt"/>
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/Example1.java"/>
           <param name="type" value="config"/>
         </macro>
         <p id="Example1-code">Example:</p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/Example1.txt"/>
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/Example1.java"/>
           <param name="type" value="code"/>
         </macro>
         <p id="Example2-config">
           To configure the check for fields only:
```

Comment:
Please move Example2 under example with `name="validateOnlyOverlapping" value="false"` .

We need change description to `To configure the check demand all fields only to have 'this.'`

To keep same description, we need to update config to have `name="validateOnlyOverlapping" value="false"` 

Only in this case such example can make sense, a bit outdated style demand, but it is what it is.

Reply:
Done.